packages/react-shopify-app-route-propagator/CHANGELOG.md

```diff
@@ -6,6 +6,8 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 
 ## [Unreleased]
 
+This package is now part of `@shopify/app-bridge-react`, starting from version 1.17.0
```

Comment:
huh, you probably wont need this since its in the README anyway. But since its debatable its up to you!
(if you want, it can be deleted in the lerna release process)